tests/e2e/support/helpers.js

```diff
@@ -8,11 +8,16 @@ export const opponentPassword = 'deviousTrickery';
  * @param {boolean} alreadyAuthenticated: skips setup steps: db wipe, signup, navigate /
  */
 export function setupGameAsP0(alreadyAuthenticated = false, isRanked = false) {
-  if (!alreadyAuthenticated) {
-    cy.wipeDatabase();
-    cy.visit('/');
-    cy.signupPlayer(username, validPassword);
-  }
+  // This is absolutely wild but we need to wrap these calls in a cy.then to force
+  // the logic to fire in the proper order under the hood
+  // https://docs.cypress.io/api/commands/then
+  cy.then(() => {
+    if (!alreadyAuthenticated) {
+      cy.wipeDatabase();
+      cy.visit('/');
+      cy.signupPlayer(username, validPassword);
+    }
+  });
```

Comment:
THIS. THIS FIXES THE FORBIDDEN ERRORS. It was a race condition after all and wrapping them in a cy.then forces them to resolve first before moving on to the create function.